Resolved bug 1353 : Design System > Components > Tenant Information > Default > After submitting field not clear.

nitinsonawane123 opened the pull request:
## Description
> Resolve the issues on Components: 
1. Resolved issue on Components - Scope Basic Resource - Default - Check box text spacing issue.
2. Resolved issue on Components - Role - Default - Check box text alignment & spacing issue.

## Screenshots :
 
![image](https://github.com/user-attachments/assets/d90a1784-1bc7-4636-984b-55aa70fb8336)
![image](https://github.com/user-attachments/assets/d9a99cb7-d15d-4262-b842-f9ed090e005b)

 
## Checklist:
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

nitinsonawane123 committed:
Commit Changes for bug 1353

nknikhilkasar committed:
Merge branch 'development' into users/nitin/bug-1353-component-tenant-information-after-submitting-field-not-clear

nknikhilkasar commented:
pls add proper password validation and show password icon for password field
![nitinbug111](https://github.com/user-attachments/assets/2e34eafd-dab9-426b-b423-b254fafe94d8)


also add proper URL validation for database URL
![nitinbug22](https://github.com/user-attachments/assets/ab3bca9e-3f6d-4c99-8745-78ec9340cb7e)

nitinsonawane123 committed:
Commit Changes for bug 1353

nitinsonawane123 committed:
Merge branch 'development' into users/nitin/bug-1353-component-tenant-information-after-submitting-field-not-clear

nitinsonawane123 commented:
Resolved issue:
- Added Proper validation for password.
- show password icon for password field is not showing.
- Added proper URL validation for database URL.

![image](https://github.com/user-attachments/assets/2290119c-638a-4609-b005-c68d6ecf1a99)
![image](https://github.com/user-attachments/assets/dcc67b1c-d94c-430d-900e-7c603cb440ee)
![image](https://github.com/user-attachments/assets/8aa89964-22b4-4b78-ad4e-4e38c51787d8)

supriyartambewaiin commented:
URL should not take like this
![image](https://github.com/user-attachments/assets/be9a331a-b214-496a-a34c-60d9d6e2f5e2)

It should be taken like this
![image](https://github.com/user-attachments/assets/200114a5-5422-4168-b12c-1fd6dd3cf367)

nitinsonawane123 committed:
Merge branch 'development' into users/nitin/bug-1353-component-tenant-information-after-submitting-field-not-clear

nitinsonawane123 committed:
Commit Changes for bug 1353

nitinsonawane123 committed:
Merge branch 'development' into users/nitin/bug-1353-component-tenant-information-after-submitting-field-not-clear

nitinsonawane123 commented:
Resolved.
- Added proper URL validation for database URL.

![image](https://github.com/user-attachments/assets/8f568ec9-305a-44e0-b9dc-fc26e0b0483f)
![image](https://github.com/user-attachments/assets/a3259979-c58f-40a0-bf38-d0802c0619fe)
![image](https://github.com/user-attachments/assets/54f7948f-bfef-4bc3-84d9-e704ff5e0525)

supriyartambewaiin reviewed (approved):
Approved
